_drafts/randy/2022-07-17-10-second-video.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "How Can 10 Second of Video Take All Day?"
+author: randy
+permalink: /@randy/10-second-video-takes-all-day
+image: "assets/img/10-second-video.png"
+description: In the age of Tik-Tok and flavor-of-the-month mob apps that instantly share minute insights to the world, a more complex video will take more time to produce. Here is a creative project review of our own making.
+tags: tails randy analog-sampling
+---
+
+![How Can 10 Seconds of Video Take All Day?](/assets/10-second-video.png)
```

Comment:
This can be removed as it is correctly used line 6 as the preview image for the article

Reply:
done removed